cmd/root.go

```diff
@@ -71,35 +72,35 @@ var RootCmd = &cobra.Command{
 		}
 		glog.V(1).Infof("Running checks for benchmark %v", bv)
 
-		if isMaster() {
-			glog.V(1).Info("== Running master checks ==")
-			runChecks(check.MASTER, loadConfig(check.MASTER, bv))
-
-			// Control Plane is only valid for CIS 1.5 and later,
-			// this a gatekeeper for previous versions
-			valid, err := validTargets(bv, []string{string(check.CONTROLPLANE)}, viper.GetViper())
-			if err != nil {
-				exitWithError(fmt.Errorf("error validating targets: %v", err))
-			}
-			if valid {
-				glog.V(1).Info("== Running control plane checks ==")
-				runChecks(check.CONTROLPLANE, loadConfig(check.CONTROLPLANE, bv))
-			}
+		isMasterNode := isMaster()
```

Comment:
Can you explain your thinking on changing this logic here? I think I preferred all the decisions about whether to skip a section here in one place, but perhaps I am missing something?